packages/cli/commons/src/errors/cliBaseError.ts

```diff
@@ -43,6 +43,10 @@ export class CLIBaseError extends Error {
         color = yellow;
         break;
 
+      case SeverityLevel.Info:
+        color = cyan;
+        break;
+
       default:
```

Comment:
default and Error are the same, this could be more easily reflected with
```
default:
case SeverityLevel.Error:
  color = red;
  break;
```

And also: DRY